test/autorest/generated/bytegroup/models.go

```diff
@@ -13,10 +13,10 @@ import (
 // ByteArrayResponse is the response envelope for operations that return a []byte type.
 type ByteArrayResponse struct {
 	// RawResponse contains the underlying HTTP response.
-	RawResponse *http.Response
+	RawResponse *http.Response `json:"-"`
```

Comment:
These tags aren't necessary as the result isn't sent in an envelope (remember, the envelope is ARM-specific).  Same applies to all models.go here.

Reply:
So I added a condition to only add tags in the case that the open-api type we receive is ARM, but since the default is ARM even when not specified, all of this codegen defaults to ARM and adding the tags. :/